data/communities.json

```diff
@@ -136,5 +133,110 @@
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/PyLadiesLondon"
+    }
+  ],
+  "codeOfConduct": "http://www.pyladies.com/CodeOfConduct/",
+  "levels": ["beginner", "intermediate", "advanced"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  },
+  {
+  "name": "Clojure Bridge",
+  "description": "ClojureBridge aims to increase diversity within the Clojure community by offering free, beginner-friendly Clojure programming workshops for underrepresented groups in tech.",
+  "url": "http://www.clojurebridge.org",
+  "logo": "https://avatars3.githubusercontent.com/u/5693478?v=3&s=400",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/ClojureBridgeLN"
+    }
+  ],
+  "codeOfConduct": "https://bridgefoundry.org/code-of-conduct/",
+  "levels": ["beginner"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  },
+  {
+  "name": "Node Girls",
+  "description": "Node Girls runs free JavaScript and Node.js workshops for women, non-binary and trans folk around the world.",
+  "url": "http://nodegirls.io/london.html",
+  "logo": "http://nodegirls.io/images/ng-logo.svg",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/nodegirlslondon"
+    }
+  ],
+  "codeOfConduct": false,
+  "levels": ["beginner", "intermediate"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  },
+  {
+  "name": "Rails Girls",
+  "description": "We are making technology more approachable for women. Workshop participants can learn sketching, prototyping, basic programming and get introduced to the world of technology.",
+  "url": "http://railsgirls.london",
+  "logo": "http://railsgirls.london/assets/logo-d9e82a4b8937e091246b8390a09095a915fa36bc7635746be9243a440c2e1dca.svg",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/railsgirls_ldn"
+    }
+  ],
+  "codeOfConduct": false,
+  "levels": ["beginner"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  },
+  {
+  "name": "R Ladies",
+  "description": "https://rladies.org",
+  "url": "https://www.meetup.com/rladies-london/",
+  "logo": "https://secure.meetupstatic.com/photos/event/8/6/5/7/global_456634391.jpeg",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/RLadiesLondon/"
+    }
+  ],
+  "codeOfConduct": false,
+  "levels": ["beginner", "intermediate", "advanced"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  },
+  {
+  "name": "Women Who Go",
+  "description": "We are women of the Go community",
+  "url": "https://www.meetup.com/Women-Who-Go-London/",
+  "logo": "https://secure.meetupstatic.com/photos/event/e/2/f/f/global_457978111.jpeg",
+  "social": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/womenwhogolndn/"
+    }
+  ],
+  "codeOfConduct": false,
+  "levels": ["beginner", "intermediate", "advanced"],
+  "tags": ["Code", "Women", "Inclusion", "Diversity"]
+  },
```

Comment:
The final object in the collection (array) should not end with a `,`. Line `241` should be changed from `},` to just the closing bracket `}`.